### DISCUSSION TITLE
請問語系怎麼設置為英文

### DISCUSSION THREAD

**Discussion by stan88** [zouyaoji/vue-cesium#360] [Q&A]
您好,
請問怎麼透過CDN的方式改換成英文?
透過VcConfigProvider改en-US變成下方的圖
謝謝

![2023-02-22_084848](https://user-images.githubusercontent.com/5283322/220492351-71dc885f-c1de-46fb-827f-4ace79cf3ba4.png)

**Comment by zouyaoji** [comment]
https://zouyaoji.top/vue-cesium/#/zh-CN/component/vc-config-provider

![image](https://user-images.githubusercontent.com/24430116/220507867-f7adc477-eabc-4c2b-bf65-2b6a470851c1.png)

**Comment by stan88** [comment]
可以了,感謝
